### PR TITLE
add argument na_name

### PR DESCRIPTION
**Pull request by wangben718**
related to https://github.com/Merck/metalite.ae/issues/101

### CODE REVIEW

**Review by nanxstats — CHANGES_REQUESTED** [review]
I would let @elong0527 comment on the logic. Style-wise:

1. For better clarity, how about we name the argument as `na` instead of `na_name` to be consistent with [readr](https://readr.tidyverse.org/reference/read_delim.html)? Alternatively, it could be `na_label`. It's just a bit hard to imagine what `na_name` means.

The documentation should be updated accordingly:

```r
#' @param na A character string used to label missing values. Defaults to `"Missing"`.
```

2. You should also move this argument before `use_na` in the function definition, which would make more make sense.

**Review by fb-elong — APPROVED** [review]

File: R/collect_n_subject.R [hunk 1]
```diff
@@ -38,14 +39,16 @@
 #' n_subject(r2rtf_adae$USUBJID, r2rtf_adae$TRTA)
 #' n_subject(r2rtf_adae$USUBJID, r2rtf_adae$TRTA, r2rtf_adae$SEX)
 #' n_subject(r2rtf_adae$USUBJID, r2rtf_adae$TRTA, r2rtf_adae$SEX, use_na = "always")
+#' n_subject(r2rtf_adae$USUBJID, r2rtf_adae$TRTA, r2rtf_adae$SEX, na = "Null")
```
Comment (LittleBeannie) [inline R/collect_n_subject.R:42]: It looks like `na =...` is not a defined argument...?

**Review by LittleBeannie — APPROVED** [review]

File: R/collect_n_subject.R (hunk 1 above)
Comment (wangben718) [reply R/collect_n_subject.R:42]: I just change it from the function argument.

**Review by nanxstats — APPROVED** [review]
Looks flawless!

**Pull request merged**